Prace_domowe/Praca_domowa3/Wolny_Krzysztof/PD3.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>Regresja logarytmiczna to jakiś nowy algorytm? \:p</p><p> </p><p>Przydałoby się napisać które to są hyperparametry, z których korzystasz i do czego służa</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>